### ISSUE TITLE
Nginx Error NanoExpress Server

### ISSUE THREAD

**Issue by meteorhr** [nanoexpress/nanoexpress#649]
I've created two simple servers.

Version
Nginx - 1.24.0
NanoExpress - 6.2.2
Express Classic -4.18.2

NanoExpress 

``` 
import nanoexpress from 'nanoexpress';

const app = nanoexpress();

app.get('/', (req, res) => { return res.send({ status: 'ok' }); });

app.listen(3000);
 ```

Express Classic 

```
import express from "express";

const app = express(); const port = 3000;

app.get('/', (req, res) => { res.send('Welcome to my server!'); });

app.listen(port, () => { console.log(`Server is running on port ${port}`); });

``` 
In the Nginx settings

`proxy_pass http://localhost:3000;`

When querying the curl server http://localhost:3000 both servers work fine.

When requesting through nginx, the server on express classic works fine. Server on NanoExpress gives error 502 and makes an entry in the logs:

`upstream prematurely closed connection while reading response header from upstream, client: 2.135.64.190, server: api.meteorhr.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "api.meteorhr.com", referrer: "http://api.meteorhr.com/`

What is missing in nginx for correct work of NanoExpress?

**Comment by meteorhr** [comment]
In case anybody else is facing this issue, I solved it by adding proxy_http_version 1.1; in the location block.

**Comment by meteorhr** [comment]
In case anybody else is facing this issue, I solved it by adding proxy_http_version 1.1; in the location block.
